bitrot.go

```diff
@@ -219,6 +225,37 @@ func (cmd *Compare) Execute(args []string) (err error) {
 	return nil
 }
 
+func (cmd *CompareLatestManifests) Execute(args []string) (err error) {
+	assertNoExtraArgs(&args, cmd.logger)
+	oldPath := pathString(cmd.Arguments.Old)
+	newPath := pathString(cmd.Arguments.New)
+
+	oldManifest := LatestManifestFileForPath(oldPath)
+	newManifest := LatestManifestFileForPath(newPath)
+
+	if oldManifest == nil {
```

Comment:
I can, but the overhead/boilerplate is making me hesitate. I think the best compromise might be to make the formatting string a constant, to at least remove the string literal duplication.